raiden/message_handler.py

```diff
@@ -47,19 +45,15 @@ def __init__(self, raiden):
 
     def on_message(self, message, msghash):  # noqa pylint: disable=unused-argument
         """ Handles `message` and sends an ACK on success. """
-        if log.isEnabledFor(logging.INFO):
-            log.info('message received', message=message)
-
         cmdid = message.cmdid
 
-        # using explicity dispatch to make the code grepable
-        if cmdid == messages.ACK:
-            pass
-
-        elif cmdid == messages.PING:
-            pass
+        # ack and ping messages are not forwarded to the handler
```

Comment:
You can just remove the commented out code and leave the initial comment.

Reply:
done